apps/emqx_management/test/emqx_mgmt_api_clients_SUITE.erl

```diff
@@ -86,8 +87,11 @@ t_clients(_) ->
             timeout
         end,
     ?assertEqual(ok, Kick),
-    AfterKickoutResponse2 = emqx_mgmt_api_test_util:request_api(get, Client2Path),
-    ?assertEqual({error, {"HTTP/1.1", 404, "Not Found"}}, AfterKickoutResponse2),
+    %% Client info is cleared after DOWN event
+    ?retry(_Interval = 100, _Attempats = 5, begin
+        AfterKickoutResponse2 = emqx_mgmt_api_test_util:request_api(get, Client2Path),
+        AfterKickoutResponse2 =:= {error, {"HTTP/1.1", 404, "Not Found"}}
```

Comment:
I think this needs to be an assert for this to work.  `?retry` doesn't check the return value.